server/core/config/base.py

```
@@ -14,6 +14,10 @@
 from pathlib import Path
 from dotenv import load_dotenv
 
+# third party imports
+from .celery_config import *
```

Ruff (0.8.0)

`from .celery_config import *` used; unable to detect undefined names

(F403)